VTEX - Marketplace Protocol - External Marketplace Mapper.json

```diff
@@ -897,373 +1001,202 @@
                                                     },
                                                     "isCertified": {
-                                                        "title": "isCertified",
-                                                        "description": "Demonstrates Demonstrates if the connector is certified.",
-                                                        "type": "boolean",
-                                                        "example": true
+                                                        "description": "Indicates whether the connector is certified.",
+                                                        "type": "boolean"
                                                     },
                                                     "contactInfo": {
-                                                        "title": "contactInfo",
                                                         "description": "Seller contact information.",
                                                         "type": "object",
-                                                        "required": [
-                                                            "name",
-                                                            "email",
-                                                            "phone",
-                                                            "defaultMessage"
-                                                        ],
                                                         "properties": {
                                                             "name": {
-                                                                "title": "name",
                                                                 "description": "Name of the person responsible for the marketplace account interested in connecting to the merchant.",
-                                                                "type": "string",
-                                                                "example": "Jhon Jones"
+                                                                "type": "string"
                                                             },
                                                             "email": {
-                                                                "title": "email",
-                                                                "description": "Email of the person responsible for the marketplace account interested in connecting to the merchant.",
-                                                                "type": "string",
-                                                                "example": "comersialsp@comercialsaopaulo.com"
+                                                                "description": "Email address of the person responsible for the marketplace account interested in connecting to the merchant.",
```

Comment:
Alterar termo de se conectar ao merchant.

Reply:
Não entendi

Reply:
Especialmente neste contexto de uso de conector, seria bom utilizar outra expressão que não "connecting to the merchant". Não se conecta ao merchant propriamente.

EX: "merchant's account"